tests/longhaul/base_block_test.go

```diff
@@ -99,24 +94,35 @@ func performBlockOperations(installer *installer.InstallHelper, db *utils.MySQLH
 	var wg sync.WaitGroup
 	for i := 1; i <= installer.Env.LoadConcurrentRuns; i++ {
 		wg.Add(1)
-		go dbOperation(db, &wg, installer.Env.LoadTime)
+		go dbOperation(db, &wg, installer.Env.LoadTime, installer.Env.LoadSize)
 	}
 	wg.Wait()
 }
 
-func dbOperation(db *utils.MySQLHelper, wg *sync.WaitGroup, runtime int) {
+func dbOperation(db *utils.MySQLHelper, wg *sync.WaitGroup, runtime int, loadSize string) {
 	defer wg.Done()
+	ds := 100000
+	switch strings.ToUpper(loadSize) {
+	case "S":
+		ds = 105000 //.1M * 5 columns * 6 = 3M per thread
```

Comment:
i'd suggest letting the compiler compute these numbers for you.

Reply:
i tried that but in  sizeof just kept giving size varriable not size of string or data the varaible is referencing to. So ended up using number of chars instead.